docs/preview_conversion_analyzer.md

```diff
@@ -0,0 +1,129 @@
+do not get copied to docs.microsoft.com; they're just an aid to reading this spec. -->
+
+<!-- If you're modifying an existing API, included a link here to the
+existing page(s) -->
+
+<!-- For example, this section is a place to explain why you're adding this API rather than
+modifying an existing API. -->
+
+<!-- For example, this is a place to provide a brief explanation of some dependent
+area, just explanation enough to understand this new API, rather than telling
+the reader "go read 100 pages of background information posted at ...". -->
+
+
+## Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+This tool assists with the conversion process by providing Code Analyzers/Fixes to convert existing WinUI2 C# projects to WinUI3.
+
+### What It Does
+
+- Analyze/Convert C# files
+- Updates Namespaces for Xaml Types from `Windows.UI.Xaml` to `Microsoft.UI.Xaml`
+    - `Windows.UI.Xaml` is now `Microsoft.UI.Xaml`, so usings and explicit namespaces need to be updated
+- Converts `App.OnLaunched` Method
+    - 2 Updates need to be made to the `App.OnLaunched` method when converting to WinUI3
+    1. Target `Microsoft.UI.Xaml.LaunchactivatedEvenArgs` as the method parameter type
+    2. Instances of the parameter name in the `App.OnLaunched` method body must invoke `UWPLaunchActivatedEventArgs`
+- Highlights Deprecated Types
+    - Some Types such as `Windows.UI.Input.Inking`, and `Windows.UI.Xaml.Media.AcrylicBackgroundSource` are not supported in WinUI3. These may be identified by the analyzer as deprecated but cannot be converted automatically.
+
+### What It Does Not Do
+
+ - Resolve conflicting packages
+ - Remove Deprecated Code
+ - Analyze/Convert .xaml files
+
+## Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+### Convert Namespaces:
+Before Convert :
+
+![Visual Studio Lightbulb Suggestion](images\namespaceBefore.png#thumb)
+
+After:
+
+![Visual Studio Lightbulb Suggestion](images\namespaceAfter.png#thumb)
+
+### Convert App.OnLaunched Method:
+Before Convert :
+
+![Visual Studio Lightbulb Suggestion](images\onlaunchedBefore.png#thumb)
+
+After:
+
+![Visual Studio Lightbulb Suggestion](images\onlaunchedAfter.png#thumb)
+
+
+## Conversion Process
+<!-- Explanation and guidance on how to use the converter that doesn't fit into the Examples section. -->
+
+1. Install Microsoft.WinUI and WinUIConvert NuGet packages in your app using the NuGet package manager: see the [Getting Started](https://docs.microsoft.com/uwp/toolkits/winui/getting-started) with the Windows UI Library page for more information.
```

Comment:
This Getting Started link points to WinUI2

Reply:
updated to https://docs.microsoft.com/en-us/windows/apps/winui/winui3/#install-winui-3-preview-2